.gitignore

```diff
@@ -72,4 +72,7 @@ CMakeLists.txt.user*
 mocs*
 kxml_compiler/kxml_compiler*
 
+# cmake generates Codeblocks project???
```

Comment:
Depends on the generator being used (-G command-line argument).
Old QtCreator required that, but this is no longer the case.

Reply:
Ah I left it in accidentally (was quite surprised when first saw this artifact), I can move it to an another PR if necessary.